choppa/IO/input.py

```diff
@@ -218,21 +227,35 @@ def extract_ligands(system):
         """[Placeholder] Returns a system's ligands"""
         return system
 
-    def check_validity(self, complex):
+    def reset_complex_sequence(self, complex):
+        """Adjusts protein sequence indexing to run from 1 to n, rather than whatever wonky indexing
+        the crystallographer may have come up with. BioPython can do this but there are some
+        protections built in against hard re-indexing multi-chain proteins. By first setting
+        the indexing super high (starting at 100,000) and then re-indexing starting from 1 we can
+        circumvent these protections. For more details see https://github.com/biopython/biopython/pull/4623
         """
-        [Placeholder] Does some quick checks to make sure the imported PDB structure is valid. We're
-        not doing any kind of protein prep, just whether biopython _is able to_ read the PDB
-        file and we try to figure out what entry names the solvent/ligands have (if there are any)
-        """
-        return complex
+        # first set indexing to an unphysically high number
+        original_index = []
+        residue_N = 100000
```

Comment:
I would make this number max(residue_numbers + 1) to avoid some kind of cutoff artefact in the extremely unlikely event someone passes in a megaprotein, probably unnecessary but good practice.